docs/capabilities/faq-security.md

```diff
@@ -40,6 +44,10 @@ Open Source Health offers the option to create an SBOM (Software Bill Of Materia
 
 SIG gathers data from multiple ecosystems. Most of these ecosystems link library vulnerabilities to CVEs, but some provide their own data that is not connected to CVEs.
 
+### What are the risk thresholds that determine low-medium-high risk?
```

Comment:
Maybe we can write instead: "What are the risk thresholds for determining whether a risk is low-medium-high?"

Feels a bit less repetitive with the two risk words there, what do you think? :)